src/os/ui/alert/alertpopup.js

```diff
@@ -238,6 +238,24 @@ os.ui.alert.AlertPopupCtrl.prototype.displayAlert_ = function(event) {
     dismissDispatcher.listenOnce(os.alert.AlertEventTypes.DISMISS_ALERT, dismiss, false, this);
 
     var message = event.getMessage();
+
+    // Check if message has a url and if it does adds a hyperlink
+    var match = message.match(os.url.URL_REGEXP_LINKY);
+    if (match) {
+      var html = [];
+      var url = match[0];
+      var i = match.index;
+      html.push(message.substring(0, i));
+      html.push('<a ');
+      html.push('href="');
+      html.push(url);
+      html.push('">');
+      html.push(os.ui.escapeHtml(message.substring(i, i + match[0].length)));
+      html.push('</a>');
+      message = message.substring(i + match[0].length);
+      html.push(message);
+      message = html.join('');
+    }
     var popup = {
       'id': id,
       'msg': goog.string.truncate(message, 500),
```

Comment:
So, it looks like the message gets truncated once it is over 500 characters.  I wonder if it will cause issues if it gets truncated in the middle of your link?

If it is an issue, maybe add a check where if it will get truncated you don't add the link?